From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 12 Nov 2021 20:25:22 +0000
Subject: [PATCH] Fix creation of temp build dir

Since `$source_dir` is just a symlink to the actual source directory,
using `cp -a` on it just copies the symlink instead of the contents.

This makes the build fail when the source directory isn't writable.

Commit d038a604bf525b01d7bd12b98706b1f861e4ea27 broke this when it
dropped the trailing slash.
---
 dkms.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dkms.in b/dkms.in
index c907d1f60a61..115eb062a382 100644
--- a/dkms.in
+++ b/dkms.in
@@ -872,7 +872,7 @@ prepare_build()
 
     # Set up temporary build directory for build
     rm -rf "$build_dir"
-    cp -a "$source_dir" "$build_dir"
+    cp -a "$source_dir/" "$build_dir"
 
     cd "$build_dir"
 
